kinder/pkg/cri/containerd/alterhelper.go

```diff
@@ -36,9 +37,6 @@ func GetAlterContainerArgs() ([]string, []string) {
 	runArgs := []string{
 		// privileged is required for "ctr image pull" permissions
 		"--privileged",
-		// the snapshot storage must be a volume.
-		// see the info in Commit()
-		"-v=/var/lib/containerd",
```

Comment:
Removing this line  is the core of this change, because with "-v=/var/lib/containerd" all the pre-loaded images were lost.
everything else is simply restoring the workflow

Reply:
ack, thanks for the info.